common/beerocks/bwl/dummy/sta_wlan_hal_dummy.cpp

```diff
@@ -41,6 +41,8 @@ bool sta_wlan_hal_dummy::connect(const std::string &ssid, const std::string &pas
     return true;
 }
 
+bool sta_wlan_hal_dummy::associate(const std::string &bssid, uint8_t channel) { return true; }
```

Comment:
I think it's useful to already add a log message in each of the implementations, even if it's currently unimplemented.

Reply:
added debug prints